_includes/start_in_docker/v2.0/mac-linux-steps.md

```diff
@@ -0,0 +1,167 @@
+## Before You Begin
+
+Make sure you have already [installed the official CockroachDB Docker image](install-cockroachdb.html).
```

Comment:
Should this say "the unstable CockroachDB Docker image"?

Reply:
Is it ok to accept this slightly misleading text for the sake of not doing something unnecessarily complicated with variables?

Reply:
Seems ok to me.